tests/pipelines/main.nf.test

```diff
@@ -51,6 +51,24 @@ nextflow_pipeline {
             def expected_arborator_meta_included = path("$baseDir/tests/data/arborator/basic/metadata.included.tsv")
             assert actual_arborator_meta_included.text == expected_arborator_meta_included.text
 
+            assert path("$launchDir/results/arborator/1_clusters.tsv").exists()
```

Comment:
Thanks for adding these. Could you add assert statements to check that these exist in the `iridanext.output.json` file too? I have found that some of these files aren't being included.

Reply:
Added in f3b077da6dfe30af0c6eaf75f975939dea072525